### PR TITLE
[1787] Update allocations interruption screen content for 2022-2023

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

Updating the content on the interruptions screen in preparation for the next cycle.

### Guidance to review

- Navigate to `/rollover-recruitment`

### Screenshot

image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
- [x] Product Review

### CODE REVIEW

**Comment by avinhurry** [comment]
[Update allocations interruption screen content](https://trello.com/c/ke4GIqeG/1787-update-allocations-interruption-screen-content)

File: app/views/pages/rollover_recruitment.html.erb [hunk 1]
```diff
@@ -3,19 +3,13 @@
...
-      You do not need to request permission to recruit for any courses other
-      than fee-funded PE. Accredited bodies should request fee-funded PE by
-      <%= Settings.allocations_close_date.to_s(:govuk) %>.
-    </p>
-
-    <p class="govuk-body">
-      You can publish all other courses without requesting permission. This
-      applies to courses copied over from the current cycle and new courses
-      added for <%= next_recruitment_cycle_period_text %>.
+      You are now able to request permission to recruit for fee-funded PE for the 2022 to 2023 cycle.
```
Comment (AbigailMcP) [inline app/views/pages/rollover_recruitment.html.erb:10]: `2022 to 2023` should use our new helper rather than being hardcoded.

**Review by AbigailMcP — APPROVED** [review]

**Review by defong — APPROVED** [review]
Looks good,


Just need to fix the hardcoded bit.

image

**Review by AbigailMcP — APPROVED** [review]

**Comment by gregknight1** [comment]
@avinhurry Just one change: Can you separate each sentence into its own paragraph, please. So it will appear like this:

**_You are now able to request permission to recruit for fee-funded PE for the 2022 to 2023 cycle.

You do not need to request permission to recruit in Publish for any courses other than fee-funded PE. 

Accredited bodies should request fee-funded PE by 2 July 2021._**

Just makes it easier to read and digest all the info. Thanks.

**Pull request merged**